orchard_merkle_tree.py

```diff
@@ -32,9 +73,253 @@ def merkle_crh(layer, left, right):
 assert merkle_crh(MERKLE_DEPTH - 1 - 25, left, right) == parent
 assert merkle_crh(MERKLE_DEPTH - 1 - 26, left, right) != parent
 
-def empty_roots():
-    empty_roots = [UNCOMMITTED_ORCHARD]
-    for layer in range(0, MERKLE_DEPTH)[::-1]:
-        bits = i2lebsp(L_MERKLE, empty_roots[-1].s)
-        empty_roots.append(merkle_crh(layer, bits, bits))
-    return empty_roots
+def main():
+    args = render_args()
+
+    bundle_commitments = [
```

Comment:
This should not be in this file (this is where #36 will go). These should go into a separate script file (they aren't really what this repo is for, but there isn't any better place to put them).

Reply:
Document how these were generated, in case they need to be regenerated.